characterisation/exceptions/HO200113.test.ts

```diff
@@ -0,0 +1,42 @@
+import World from "../../utils/world"
+import { processPhase2Message } from "../helpers/processMessage"
+import { asnPath } from "../helpers/errorPaths"
+import generatePhase2Message from "../helpers/generatePhase2Message"
+import MessageType from "../types/MessageType"
+import { ResultClass } from "../types/ResultClass"
+
+describe.ifPhase2("HO200113", () => {
+  afterAll(async () => {
+    await new World({}).db.closeConnection()
+  })
+
+  it.each([MessageType.ANNOTATED_HEARING_OUTCOME, MessageType.PNC_UPDATE_DATASET])(
+    "creates a HO200113 exception for %s when NEWREM exists, there are no remand CCRs, and SENDEF exists",
```

Comment:
There are two places within `validateOperations` that a HO200113 can occur.

I believe this test covers: https://github.com/ministryofjustice/bichard7-next-core/blob/3200a2064d313589cdbb71d161732d0efc773bbb/packages/core/phase2/lib/getOperationSequence/validateOperations.ts#L41-L43

but what about: https://github.com/ministryofjustice/bichard7-next-core/blob/3200a2064d313589cdbb71d161732d0efc773bbb/packages/core/phase2/lib/getOperationSequence/validateOperations.ts#L97-L99?

Have we checked this is possible or not?